# Double: max cross track tolerance error
# Units: meter
max_crosstrack_error: 25

# Double: downtrack_range
# Units: meter
destination_downtrack_range: 0